examples/react-datastore/src/hooks.js

```diff
@@ -45,9 +46,9 @@ export const useFindTodos = () => {
         })();
 
         const subscriptions = [];
-        subscriptions.push(TodoModel.on("ADD", refreshState));
-        subscriptions.push(TodoModel.on("UPDATE", refreshState));
-        subscriptions.push(TodoModel.on("DELETE", refreshState));
+        subscriptions.push(TodoModel.on(DatabaseEvents.ADD, refreshState));
```

Comment:
DataEvents looks cooler to me but it is ok to have this name

Reply:
I thought you said "DatabaseEvents" in meeting.

Reply:
No worries. We will review all namings before release